artacle-swagger.yaml

```diff
@@ -918,6 +1194,202 @@ components:
               nullable: true
               example: "0x9df947cd8a9810964473339c38a95201ecbe112d"
 
+    CollectionTraits:
+      type: object
+      required:
+        - traitValue
+        - valuationAllTime
+        - valuationAllTimeUSD
+        - valuationRecent
+        - valuationRecentUSD
+        - recentPeriodMonths
+        - totalAmountOfSales
+        - medianTTFAllTime
+        - medianTTFRecentPeriod
+        - quantity
+        - largeDistinctCount
+      properties:
+        traitValue:
```

Comment:
We do use inconsistent notation here: we accept name and value separately in the input, and concatenate them in the output. I think that the output should provide name and value separately